test/snippet/argument_parser/validators_input_file.cpp

```diff
@@ -5,16 +5,17 @@
 
 int main(int argc, const char ** argv)
 {
-    seqan3::argument_parser myparser{"Test", argc, argv}; // initialize
+    seqan3::argument_parser myparser("Test", argc, argv); // initialize
```

Comment:
construction should use braces not parentheses?

EDIT: this should be the case for all the constructors in the snippets of argument parser and validators?

Reply:
Why making an exception for the argument parser class? 😕

Reply:
huh? no exception. I am asking why you are chaning braces to parentheses? I thought we always construct using braces?

Reply:
Yeah ok I kind of misinterpreted the comment as a suggestion from you. 🤕 
I have another commit changing all instances of the parentheses in the argument parser doc and tests.